assets/masandra.js

```diff
@@ -0,0 +1,58 @@
+$('#getStarted').click(function (e) {
+    e.preventDefault();
+    console.log("I Work");
+    $('#ingredientSection').attr("style", "display:block");
+    $('#introCard').attr("style", "display:none");
+    localStorage.clear();
+});
+
+$('#goButton').click(function (e) {
+    e.preventDefault();
+    displayIngredient();
+});
+
+$('#clearList').click(function (e) {
+    e.preventDefault();
+    localStorage.clear();
+});
+
+
+function displayIngredient() {
+        var userInputIngEl = $('#userInputIng').val();
+        var ingredientListEl = $("#ingredientList");
+        var newIngBtn = $('<li class="item button is-success"><span class="icon is-small"><i class="fas fa-check"></i></span></li>');
+        newIngBtn.appendTo(ingredientListEl);
+        newIngBtn.text(userInputIngEl);
+        newIngBtn.val(userInputIngEl);
+        saveIngredient();
+        fetchIngredients();
```

Comment:
missing closing brace I think. unless you meant to declare save ingredient as a nested function? I think that's somewhat anti-pattern. Named functions should declared at the top level file scope. nested functions should be anonymous